package.json

```diff
@@ -13,7 +13,7 @@
     "postinstall": "lerna bootstrap --stream",
     "lint": "yarn lint:css && yarn lint:js && yarn lint:markdown",
     "lint:css": "stylelint packages/*/src/**/*.js && stylelint utils/**/*.js",
-    "lint:js": "eslint packages/*/src/ utils/**/*.js --max-warnings 0",
+    "lint:js": "eslint \"packages/*/src/\" \"utils/**/*.js\" --max-warnings 0",
```

Comment:
I'd prefer the simpler `eslint packages/*/src/ utils/ ...` here. Easier to read.